packages/components/src/components/SquiggleViewer/index.tsx

```diff
@@ -70,14 +88,14 @@ export const SquiggleViewer: React.FC<Props> = ({
           ...(localSettings.chartSettings || {}),
         },
         environment: {
-          ...environment,
-          ...(localSettings.environment || {}),
+          ...env,
+          ...localSettings.environment,
```

Comment:
Huh. The environment can't really be edited in local settings. I'm confused why it's there, maybe I didn't think enough about what I was doing when I wrote that code, or maybe I intended to make `xyPointLength` configurable on chart level?..

Anyway, if you prefer to take the environment from project, then there's currently no point in tracking it through local or merged settings, all of that and all your `Omit<MergedItemSettings, "environment">` can be cleaned up, I think.